src/Scouting.elm

```diff
@@ -55,32 +61,91 @@ update msg model =
         TeamInput input ->
             { model
                 | team = String.toInt input
-                , driverStation = stationIndex (String.toInt input) model.match
+                , driverStation = stationName (String.toInt input) model.match
             }
 
         MatchInput input ->
             { model
                 | match = String.toInt input
-                , driverStation = stationIndex model.team (String.toInt input)
+                , driverStation = stationName model.team (String.toInt input)
             }
 
         Start ->
             { model
-                | isStarted = not model.isStarted
-                , driverStation = stationIndex model.team model.match
+                | isStarted = switchButtonState model
+                , driverStation = stationName model.team model.match
             }
 
 
-startButton : Bool -> Html.Html Msg
-startButton isStarted =
-    Html.div []
-        [ Html.button [ onClick Start ]
-            [ Html.text <|
-                if isStarted then
-                    "Started"
+ifCorrectInput : Model -> ButtonState
+ifCorrectInput model =
+    let
+        modelStr =
+            strModel model
+
+        isStation string =
+            model.driverStation == string
+    in
+    if modelStr.scouter == "" then
+        TriedPush
+
+    else if modelStr.team == "" then
+        TriedPush
+
+    else if modelStr.match == "" then
+        TriedPush
+
+    else if isStation " " || isStation "  " then
+        TriedPush
+
+    else
+        Pushed
+
+
+switchButtonState : Model -> ButtonState
+switchButtonState model =

```

Comment:
This name isn't very indicative.
You might as well implement the "switch"ing as `always Pushed`.